gemrb/core/Game.cpp

```diff
@@ -40,8 +40,27 @@
 #include "GUI/GameControl.h"
 #include "System/DataStream.h"
 #include "System/StringBuffer.h"
+#include <iterator>
+#include <vector>
+#include <algorithm>
 
 namespace GemRB {
+	
+	
+struct HealingResource {
+	ieResRef resref;
+	Actor* caster;
+	ieWord amount;
+};
+
+struct Injuree {
```

Comment:
You're right, I slavishly copied it from the original Game.cpp. In fact, if it were a word it'd be the one administering the injuring. I'll change it once with the other yet to be uncovered errors.